cdap-app-fabric/src/main/java/io/cdap/cdap/security/TokenSecureStoreRenewer.java

```diff
@@ -143,24 +139,22 @@ public Credentials createCredentials() {
   }
 
   private long calculateUpdateInterval() {
-    return calculateUpdateInterval(yarnConf, secureExplore);
+    return calculateUpdateInterval(yarnConf, secureToken);
```

Comment:
No need to pass `secureToken`, modify the `calculateUpdateInterval` to take only `yarnConf` as argument.